staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -130,10 +130,12 @@ func (a *MutatingWebhook) SetServiceResolver(sr config.ServiceResolver) {
 
 // SetScheme sets a serializer(NegotiatedSerializer) which is derived from the scheme
 func (a *MutatingWebhook) SetScheme(scheme *runtime.Scheme) {
+	admissionScheme := runtime.NewScheme()
```

Comment:
I thought that the schemes we used had to be passed in so that the internal objects the admission plugin gets could be reliably converted to the external representations.  Different schemes for different layers or something?

Reply:
compare the converter below. The new scheme is about the versioned AdmissionReview with encoded payload objects.